src/modules/account/views/ViewAccountDisplayAll.vue

```diff
@@ -208,10 +212,23 @@ export default {
     emitter.on('filterByLabel',e=>{
       labelNames.value = e
     })
-    
+
+    const filterQuery = ref("");
+    const filteredAccounts = computed(() => {
+    const query = filterQuery.value.toLowerCase();
+      if(filterQuery.value == ""){
+        return accounts.value;
+      }
+      return accounts.value.filter((item) =>{
+        return Object.values(item).some((word) =>
+          String(word).toLowerCase().includes(query));
```

Comment:
should filters out unrelated data, such as isMultisig and multisigInfo

Reply:
Hi Chun Lam, changed to filter only name and address. Please review again. Thanks